Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -251,7 +253,7 @@ [FromQuery] [RegularExpression(EncodingHelper.ValidationRegex)] string? subtitle
             Height = height,
             VideoBitRate = videoBitRate,
             SubtitleStreamIndex = subtitleStreamIndex,
-            SubtitleMethod = subtitleMethod ?? SubtitleDeliveryMethod.Encode,
+            SubtitleMethod = subtitleMethod ?? SubtitleDeliveryMethod.External,
```

Comment:
I think the idea is that `Encode` will always work.
But `SubtitleMethod` must be specified for stable results, though.

Reply:
The problem is that StreamInfo's URL builder always specifies Encode and will leave it blank for External. We should make the behavior consistent.